### PR TITLE
Rewrite Vue charts code

### LINKED ISSUE #5089: Rewrite Vue charts code
Because the Vue charts library does not generate a release package, every time NPM install will pull the latest code again, which will have a certain impact on the stability of the front-end UI of this project.

~~Finally decided to replace the Vue charts Library to vue-echarts-v3 library~~

### PR DESCRIPTION
**Pull request by wqzwh**
Fixes #5089

### CODE REVIEW

File: shardingsphere-ui/shardingsphere-ui-frontend/package.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     "lodash": "^4.17.11",
     "moment": "^2.24.0",
     "normalize.css": "^8.0.1",
+    "resize-detector": "^0.2.1",
```
Comment (terrymanu) [inline shardingsphere-ui/shardingsphere-ui-frontend/package.json:24]: What is this lib? can you add notice for that?

File: shardingsphere-ui/shardingsphere-ui-frontend/src/components/ChartBase/index.vue [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+<template>
+  <div class="echarts"></div>
+</template>
+
+<script>
+import echarts from 'echarts'
+import debounce from 'lodash/debounce'
+import { addListener, removeListener } from 'resize-detector'
+import Vue from 'vue'
+
```
Comment (terrymanu) [inline shardingsphere-ui/shardingsphere-ui-frontend/src/components/ChartBase/index.vue:27]: Where is this file come from? Is it develop by ourself or copy from other project?

File: shardingsphere-ui/shardingsphere-ui-frontend/package.json (hunk 1 above)
Comment (wqzwh) [reply shardingsphere-ui/shardingsphere-ui-frontend/package.json:24]: This library is the one that the official Vue version of echarts relies on, so I also quoted

File: shardingsphere-ui/shardingsphere-ui-frontend/src/components/ChartBase/index.vue (hunk 2 above)
Comment (wqzwh) [reply shardingsphere-ui/shardingsphere-ui-frontend/src/components/ChartBase/index.vue:27]: This is a secondary development, which is not completely consistent with the official code

**Comment by wqzwh** [comment]
> Why change backend notice if your code only change frontend?

The rebase operation is missing, resulting in duplicate commit

**Pull request closed without merging**